valkey.conf

```diff
@@ -2072,13 +2072,13 @@ stream-node-max-entries 100
 # server is idle the rehashing is never complete and some more memory is used
 # by the hash table.
 #
-# The default is to use this millisecond 10 times every second in order to
-# actively rehash the main dictionaries, freeing memory when possible.
+# The default is to use this millisecond 10 times every second when hz is 10,
+# in order to actively rehash the main dictionaries, freeing memory when possible.
```

Comment:
With this PR, this is no longer a "default". Before, this could be changed by changing `hz`, but with this PR it is hardcoded to use 1% of the CPU time.

So let's not use the word "default".

Instead, we can explain that the user can now change `hz` to a higher value to avoid the spikes. A higher value of `hz` will use less than time for each rehash but do it more often.